presto-orc/src/main/java/com/facebook/presto/orc/writer/SliceDirectColumnWriter.java

```diff
@@ -85,11 +86,12 @@ public SliceDirectColumnWriter(
             MetadataWriter metadataWriter)
     {
         checkArgument(column >= 0, "column is negative");
+        checkArgument(type instanceof AbstractVariableWidthType, "type is not a AbstractVariableWidthType");
```

Comment:
type is not `a` AbstractVariableWidthType -> type is not `an` AbstractVariableWidthType